src/components/EntityInfo/EntityMenu.vue

```diff
@@ -41,34 +41,6 @@
       </v-tooltip>
     </span>
 
-    <!-- Amalgamate -->
-    <span v-if="isBusiness && !isHistorical">
```

Comment:
Apparently, the design for this was very outdated (before we even implemented the More Actions dropdown).

So, this button here has been moved there and fixed tests accordingly 😁